src/views/Contacts.vue

```diff
@@ -662,6 +667,32 @@ export default {
 			this.downloadVcardPromise(vcardPromise)
 		},
 
+		/**
+		 * Open mailto: for contacts in a group
+		 *
+		 * @param {Object} group of contacts to be emailed
+		 */
+		emailGroup(group) {
+		  const emails = []
+			group.contacts.forEach(key => {
+				// The email property could contain "John Doe <john.doe@example.com>", but vcard spec only
+				// allows addr-spec, not name-addr, so to stay compliant, replace everything outside of <>
+				const email = this.contacts[key].email.replace(/(.*<)([^>]*)(>)/g, '$2').trim()
+				const name = this.contacts[key].fullName.replace(/[,<>]/g, '').trim()
+				if (email === '') {
+					return
+				}
+				if (name === null || name === '') {
+					emails.push(email)
+					return
+				}
+				emails.push(`${name} <${email}>`)
+			})
+			// We could just do mailto:${emails}, but if we want to use name-addr, not addr-spec, then we
+			// have to explicitly set the "To:" header. Plus, we want to BCC the recipients
+			window.location.href = `mailto:?bcc=${emails.map(encodeURIComponent).join(',')}`
```

Comment:
why bcc and not to?

Reply:
Email etiquette is usually to BCC group recipients to avoid leaking email addresses to the recipients. I guess if we wanted to offer an option, maybe we could hide it behind a user preference?

Reply:
I'd say this really depends. If I sent to a group of coworkers I would certainly want them to know who else go the email. For something where I send holiday wishes to clients I wouldn't want them to see each other's address.

A user preference doesn't seem right either. This is more a case-by-case decision. One email might go to undisclosed recipients, the other one is for direct recipients.

In the end most email client will allow you to move people from *to* to *bcc*. I'd vote for *to* by default and let people move recipients around if they wish.

Reply:
> I'd vote for to by default and let people move recipients around if they wish.

Would say that as well from a user perspective. The question is all about the default, and the less confusing one is if it goes in "to". :)